doc/nrf/releases/release-notes-latest.rst

```diff
@@ -435,7 +452,11 @@ The following list summarizes the most important changes inherited from upstream
       * :option:`CONFIG_OPENTHREAD_MAC_SOFTWARE_ACK_TIMEOUT_ENABLE`
       * :option:`CONFIG_OPENTHREAD_MAC_SOFTWARE_RETRANSMIT_ENABLE`
       * :option:`CONFIG_OPENTHREAD_PLATFORM_USEC_TIMER_ENABLE`
-      * :option:`CONFIG_OPENTHREAD_CONFIG_PLATFORM_INFO`
+      * :option:`CONFIG_OPENTHREAD_PLATFORM_INFO`
```

Comment:
Almost duplicating https://github.com/nrfconnect/sdk-nrf/pull/3647/files

Reply:
Unfortunately I do not understand.

Reply:
Some of these options are also added in PR https://github.com/nrfconnect/sdk-nrf/pull/3647
I don't mind where they are added, but adding them only once would probably prevent merge conflicts. ;)

Reply:
> 
> 
> Some of these options are also added in PR #3647
> I don't mind where they are added, but adding them only once would probably prevent merge conflicts. ;)

Ah... right. My bad.